projects/UOregon_Melgar.yaml

```
@@ -0,0 +1,5 @@
+Department: Cascadia Region Earthquake Science Center
+Description: "Conducting earthquake simulations as part of a larger collaboration (https://github.com/Marcus-Adair/Accelerating-Data-Intensive-Seismic-Research-Through-Parallel-Workflow-Optimization-and-Federated-CI). Planning to eventually run some ML for graph neural network GNSS denoising."
```

```suggestion
Description: >-
    Conducting earthquake simulations as part of a larger collaboration
    (https://github.com/Marcus-Adair/Accelerating-Data-Intensive-Seismic-Research-Through-Parallel-Workflow-Optimization-and-Federated-CI).
    Planning to eventually run some ML for graph neural network GNSS denoising."
```

Improving readability for wide screens (https://yaml-multiline.info/ is a lifesaver)